contributed_definitions/nyaml/NXpeak.yaml

```diff
@@ -1,59 +1,140 @@
 category: base
 doc: |
-  Description of peaks, their functional form or measured support.
+  Base class for describing spectral peaks, their functional form, and support values
+  (i.e., the discretization (points) at which the function has been evaluated).
 symbols:
   doc: |
     The symbols used in the schema to specify e.g. dimensions of arrays.
   n_support: |
-    Number of support points
+    Number of support points.
```

Comment:
Do you consider the data points within the fit region here? Is this the same as in NXbackground? 
A Sherly background can e.g. also depend on intensity outside the fitted region!

Reply:
In that case, we should probably either remove the dims or have individual dims for peaks and background (n_peak_values, n_background_values or so).